moler/cmd/unix/iperf2.py

```diff
@@ -55,7 +55,7 @@ class Iperf2(GenericUnixCommand):
 
         COMMAND_RESULT = {
             'CONNECTIONS': {
-                ("192.168.0.10", "192.168.0.12:5016"): {'report': {'Lost_Datagrams_ratio': '0%',
+                ("192.168.0.10", "5016@192.168.0.12"): {'report': {'Lost_Datagrams_ratio': '0%',
```

Comment:
General. Is doc string a good place to put example ? There are variables for examples. Variables are tested automatically with every push and doc strings are not. Every time after any change we have to remember to update doc string. Nothing happens automatically. Soon it may lead to discrepancy between real behaviour and comments.